app/code/Magento/Store/Model/Config/Placeholder.php

```diff
@@ -164,4 +206,26 @@ protected function _setValue(array &$container, $path, $value)
         }
         $currentPointer = $value;
     }
+
+    /**
+     * Set array value by path
+     *
+     * @param array &$container
```

Comment:
Could you please fix the static test failure on this line?
I guess the "&" is redundant here.

Reply:
I've copy-pasted it, now removed. Waiting for new static test results...